vdirsyncer/cli/discover.py

```diff
@@ -59,21 +59,20 @@ async def collections_for_pair(
     cache_key = _get_collections_cache_key(pair)
     if from_cache:
         rv = load_status(status_path, pair.name, data_type="collections")
-        if rv.get("cache_key", None) == cache_key:
+        if rv and rv.get("cache_key", None) == cache_key:
```

Comment:
Is this change form me and is it necessary?  The other changes here do not modify the program logic, that is they preserve all the bugs.  But this change modifies the program logic.

Reply:
This change was from your original commit.

When I reviewed this earlier the change made sense to me because of the following `elif rv`. I realise now that it is unnecessary. It should not change the logic, assuming that `rv` is never `None`.